src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -1350,12 +1341,11 @@ private function addInlineRequires(?array &$preload): string
         foreach ($lineage as $file) {
             if (!isset($this->inlinedRequires[$file])) {
                 $this->inlinedRequires[$file] = true;
-                $file = preg_replace('#^\\$this->targetDirs\[(\d++)\]#', sprintf('\dirname(__DIR__, %d + $1)', $this->asFiles), $file);
                 $code .= sprintf("\n            include_once %s;", $file);
             }
         }
 
-        return $code ? sprintf("\n        \$this->privates['service_container'] = static function () {%s\n        };\n", $code) : '';
+        return $code ? sprintf("\n        \$this->privates['service_container'] = function () {%s\n        };\n", $code) : '';
```

Comment:
this seems unrelated, right ?

Reply:
it is: we didn't generate `$this` anymore inside, but that's now back (and legit)

Reply:
https://github.com/symfony/symfony/pull/33529/files#diff-edd219a3021114261638a490a9e00ae9L93-R91

Reply:
hmm, so this was broken previously when changing the container dir ?

Reply:
possibly yes